cloudfoundry/resource_cf_user_provided_service.go

```diff
@@ -66,6 +66,11 @@ func resourceUserProvidedService() *schema.Resource {
 				ConflictsWith:    []string{"credentials"},
 				DiffSuppressFunc: structure.SuppressJsonDiff,
 			},
+			"recursive_delete": &schema.Schema{
```

Comment:
There is no recursive delete support in the UPS delete api http://apidocs.cloudfoundry.org/4.4.0/user_provided_service_instances/delete_a_particular_user_provided_service_instance.html